index.js

```diff
@@ -1,18 +1,51 @@
 #!/usr/bin/env node
+var assert = require('assert');
 require('shelljs/global');
 
 // Creates a changelog for the current build and puts it in the root
 // Commits the changelog if it updated
 // Does not push commit
 
+// The maximum number of deletions for updating the changelog should
+// be 2 lines. Typically, the "Unreleased" and "Full Changelog" links
+// are changed because the version increases.
+var MAX_DELETIONS = 2;
+
 function parseGitUri(uri) {
   return uri.match(/https:..github.com\/([^./]+)\/([^./]+).*/) ||
     uri.match(/git@github.com:(.*)\/(.*)\.git/);
 }
 
+function verifyChangelog() {
+  var diff = exec('git diff HEAD CHANGELOG.md').split('\n');
+  var changelog = cat('CHANGELOG.md').split('\n');
+  if (changelog[changelog.length - 1] === '') changelog.pop();
+  var isOneLine = (changelog.length === 1);
+  var deletions = diff.filter(/^-/.test).length;
+  var containsError = false;
+  diff.forEach(function (line) {
+    if (line.indexOf('Make a POST first') >= 0) containsError = true;
+  });
+  assert(
+    changelog.length > 1,
+    'Changelog diff should be more than 1 line long'
+  );
+  assert(
+    deletions < MAX_DELETIONS,
+    'Too many deletions (-' + deletions + '), verify that the changes to CHANGELOG.md are correct'
```

Comment:
I think I would prefer the changelog were automatically reverted, but in special circumstances we might want to increase the `MAX_DELETIONS` limit. Perhaps this should be configurable from the command line:
```
$ shelljs-changelog 10
# or
$ shelljs-changelog -d 10
# or
$ shelljs-changelog --max-deletions 10
```